onyo/lib/tests/test_commands_tree.py

```diff
@@ -59,15 +59,36 @@ def test_onyo_tree_single(inventory: Inventory,
     assert inventory.repo.git.is_clean_worktree()
 
 
+@pytest.mark.ui({'yes': True})
+def test_onyo_tree_dirs_only(inventory: Inventory,
+                             capsys) -> None:
+    r"""Display directory structure w/o any assets or other files"""
+
+    onyo_tree(inventory,
+              paths=[(str(inventory.root), inventory.root)],
+              dirs_only=True)
+    tree_output = capsys.readouterr().out
+    for path in inventory.root.rglob('*'):
```

Comment:
This test will clearly include and check for asset directories (good), but my question is: does the test repo have an asset dir in it?

I'd just like to make sure that case is tested (and there's at least a comment explicitly stating so).